core/src/main/java/nu/yona/server/messaging/entities/MessageDestination.java

```diff
@@ -5,21 +5,16 @@
 package nu.yona.server.messaging.entities;
```

Comment:
* We can do some math on the number of messages, based on [the tables created by Rine](http://wiki.yona.nu/display/DEV/Cluster+architecture). The other alternative that I still need to look in: would it be possible to use a distributed in memory database? That way, we wouldn't need to bother about caching at all.
* So it's not individually, but it's still relational. That's not the design we are applying.
* The ``MessageDestination`` is not being pushed, as that's not updated, but only the ``List`` inside it.